data/json/monsters/feral_humans.json

```
@@ -3,7 +3,7 @@
     "id": "mon_feral_human_pipe",
     "type": "MONSTER",
     "name": { "str": "feral human" },
-    "description": "Pupils dilated and what remains to be seen of the iris and sclera are bloodshot.  It still breathes but the zombies treat it like one of them.",
+    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This pipe-wielding maniac still breathes, but the zombies treat ferals like one of them.",
```

I think it's better for the description to describe how it is treated, not all others like it.
```suggestion
    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This pipe-wielding maniac still breathes, but the zombies treat it like one of them.",
```

---

Mixing pronouns is confusing. You can't be a they one sentence and an it the next.